App/Sagas/TextileSagas.ts

```diff
@@ -206,93 +197,6 @@ export function * initializeAppState () {
   yield put(TextileNodeActions.appStateChange(defaultAppState, queriedAppState))
 }
 
-export function * handleNewAppState (action: ActionType<typeof TextileNodeActions.appStateChange>) {
-  const { previousState, newState } = action.payload
-  if (previousState.match(/default|unknown/) && newState === 'background') {
-    yield * triggerCreateNode()
-  } else if (previousState.match(/default|unknown|inactive|background/) && newState === 'active') {
-    yield put(TextileNodeActions.lock(false))
-    yield * triggerCreateNode()
-  } else if (previousState.match(/inactive|active/) && newState === 'background') {
-    yield * triggerStopNode()
-  }
-}
-
-export function * triggerCreateNode () {
-  const locked = yield select(TextileNodeSelectors.locked)
-  if (locked) {
-    return
-  }
-  yield put(TextileNodeActions.lock(true))
-  yield put(TextileNodeActions.createNodeRequest(RNFS.DocumentDirectoryPath))
-}
-
-export function * triggerStopNode () {
-  yield put(TextileNodeActions.stopNodeRequest())
-}
-
-export function * createNode (action: ActionType<typeof TextileNodeActions.createNodeRequest>) {
-  const { path } = action.payload
-  try {
-    const logLevel = (__DEV__ ? 'DEBUG' : 'INFO')
-    const logFiles = !__DEV__
-    yield call(TextileNode.create, path, Config.TEXTILE_CAFE_URI, logLevel, logFiles)
-    yield put(TextileNodeActions.createNodeSuccess())
-    try {
-      const mnemonic: string = yield call(TextileNode.mnemonic)
-      yield put(PreferencesActions.updatecMnemonic(mnemonic))
-    } catch(error) {
-      // This only succeeds when the node is first created so this error is expected
-    }
-
-    yield put(TextileNodeActions.startNodeRequest())
-  } catch (error) {
-    yield put(TextileNodeActions.createNodeFailure(error))
-    yield put(TextileNodeActions.lock(false))
-  }
-}
-
-export function * startNode () {
-  const onboarded = yield select(PreferencesSelectors.onboarded)
-  if (!onboarded) {
-    yield put(TextileNodeActions.lock(false))
-    return
-  }
-
-  try {
-    yield call(TextileNode.start)
-    yield put(TextileNodeActions.startNodeSuccess())
-
-    // Restore our tokens and username
-    const tokens = yield call(TextileNode.getTokens)
-    yield put(AuthActions.getTokensSuccess(tokens))
-
-    yield put(ThreadsActions.refreshThreadsRequest())
```

Comment:
After node is started, we request that the threads be refreshed. Specifically, this gets the list of threads from the node and stores it in ThreadsRedux memory and triggers a fetch of the hashes for each thread and stores those in TextileNodeRedux memory.

Reply:
@sanderpick replied:

> You're probably doing this, but just making sure... can we call refreshMessages on the node too?

Reply:
No, at this point I'm not doing anything... I want to make sure I get a list of all the things that need to be done and exactly when they should be done. Cool, yes, we can refresh messages in the background.

Reply:
Moved to node lifecycle listener.